src/components/Button.tsx

```diff
@@ -18,11 +19,14 @@ const Button = ({
   icon = null,
   uppercase = true,
   disabled = false,
+  color,
 }: ButtonProps) => {
   return (
     <button
       className={`${
-        type === "primary" && style === "contained" && "gradient-red"
+        type === "primary" &&
+        style === "contained" &&
+        `gradient-${color && color.length > 0 ? color : "red"}`
       } ${
```

Comment:
I think you should remove all this css and put it just above in a function that returns css. Using switches, cause right now it's not really readable and scalable (even if it works)

Reply:
Good idea. can be good to fix it in a next issue